ci_framework/roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -90,3 +89,13 @@ cifmw_ceph_rgw_config:
   rgw_max_attr_size: 1024
 cifmw_cephadm_cephfs_name: "cephfs"
 cifmw_cephadm_ceph_spec_fqdn: "{{ ceph_spec_fqdn | default(false) | bool }}"
+cifmw_ceph_dashboard_spec_path: /tmp/ceph_dashboard.yml
+cifmw_cephadm_dashboard_crt: ""
```

Comment:
Same comment: if the path used to reference both `dashboard` and `grafana` are the same, we can collapse L93-L96 to just two variables, one for the key, one for the cert